src/components/Chip.test.tsx

```diff
@@ -23,4 +23,9 @@ describe("Chip", () => {
     expect(r.chip().textContent).toBe("Chip text content");
     expect(r.tooltip()).toHaveAttribute("title", "title is different");
   });
+
+  it("can set background color and color with xss prop", async () => {
+    const r = await render(<Chip text="Chip" xss={Css.lightBlue100.bgLightBlue100.$} />);
+    expect(r.chip()).toHaveStyle(`background: ${Css.lightBlue100.bgLightBlue100.$}`);
```

Comment:
I think this is a false positive. The output of `Css.lightBlue100.bgLightBlue100.$` is:
```
{
  backgroundColor: 'rgba(224,242,254,1)',
  color: 'rgba(224,242,254,1)'
 }
```

I just pulled this down and saw that you can put any value in there, and it'll pass. For example, this passes: `expect(r.chip()).toHaveStyle("background: anything");` I'm not 100% sure why that is the case. Seems like a bug based on the documentation for `toHaveStyle`.

I would suggest updating this test to:
```suggestion
    expect(r.chip()).toHaveStyle(Css.lightBlue100.bgLightBlue100.$);
```
That should work correctly.